tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/homescreen_settings.py

```diff
@@ -16,8 +16,9 @@ class HomescreenSettings(Base):
 
     _wallpaper_preview_locator = (By.CSS_SELECTOR, '.wallpaper-preview')
     _wallpaper_pick_locator = (By.CLASS_NAME, 'wallpaper-button')
-    _pick_cancel_button_locator = (By.CSS_SELECTOR,
-                                   '#action-menu-list > button[data-l10n-id="cancel"]')
+    _pick_cancel_button_locator = (By.CSS_SELECTOR, '#action-menu-list > '
```

Comment:
Strange, I changed from '#action-menu-list > button[data-l10n-id="cancel"]' because that one wasn't working.  I thought it is almost identical to '#action-menu-list button[data-l10n-id="cancel"]', I guess it only searches for the direct child, not its descendants?